code/game/atoms.dm

```diff
@@ -318,11 +320,6 @@
 	if(AM && isturf(AM.loc))
 		step(AM, turn(AM.dir, 180))
 
-GLOBAL_LIST_EMPTY(blood_splatter_icons)

```

Comment:
We no longer save a list of blood splatter icons, because they have to be remade every time the blood mixture on an object changes.